docs/source/_partials/variants-and-disabling.blade.php

```diff
@@ -27,7 +28,7 @@
 @component('_partials.customized-config', ['key' => 'modules'])
   // ...
 - {{ $utility['property'] }}: [{{$currentVariants}}],
-+ {{ $utility['property'] }}: ['responsive', 'hover', 'focus'],
++ {{ $utility['property'] }}: ['responsive', 'hover', 'active', 'focus'],
```

Comment:
You could leave this change out I think, otherwise it could become very bloated over time...

Reply:
I guess you are right